packages/Adapters/POD_C/src/DTK_NoName.h

```diff
@@ -0,0 +1,48 @@
+#ifndef DTK_NONAME_H
+#define DTK_NONAME_H
+
+#include "mpi.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Opaque pointer
+typedef void DTK_Map;
+
+// Possible data layouts:
+// * Contiguous 
+//     X1 X2 ... Xn Y1 Y2 ... Yn
+// * Interleave 
+//     X1 Y1 X2 Y2 ... Xn Yn
+typedef enum data_layout { CONTIGUOUS, INTERLEAVE } DTK_DataLayout;
+
+//----------------------------------------------------------------------------//
+// Create a DTK map operator
+DTK_Map * create_map( MPI_Comm comm,
+                      double const * src_coord,
+                      DTK_DataLayout src_layout,
+                      double const * tgt_coord,
+                      DTK_DataLayout tgt_layout,
+                      int space_dim,
+                      char const * options );
+
+//----------------------------------------------------------------------------//
+// Apply the map
+void apply_map( DTK_Map * dtk_map,
+                double const * src_field,
```

Comment:
Based on the field dimension and the number of source and target points we should know how much field data is here. Should we also make the users tell us so we can check?